daprdocs/content/en/java-sdk-docs/_index.md

```diff
@@ -0,0 +1,230 @@
+---
+type: docs
+title: "Dapr Java SDK"
+linkTitle: "Java"
+weight: 1000
+description: Java SDK packages for developing Dapr applications
+---
+
+## Pre-requisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- JDK 11 or above - the published jars are compatible with Java 8:
+    - [AdoptOpenJDK 11 - LTS](https://adoptopenjdk.net/)
+    - [Oracle's JDK 15](https://www.oracle.com/java/technologies/javase-downloads.html)
+    - [Oracle's JDK 11 - LTS](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+    - [OpenJDK](https://openjdk.java.net/)
+- Install one of the following build tools for Java:
+    - [Maven 3.x](https://maven.apache.org/install.html)
+    - [Gradle 6.x](https://gradle.org/install/)
+
+## Importing Dapr's Java SDK
+
+For a Maven project, add the following to your `pom.xml` file: 
+```java
+<project>
+  ...
+  <dependencies>
+    ...
+     // Dapr's core SDK with all features, except Actors. 
+    <dependency>
+      <groupId>io.dapr</groupId>
+      <artifactId>dapr-sdk</artifactId>
+      <version>1.2.0</version>
```

Comment:
Please, refer to 1.1.0 since that is the latest release.
Then, update the script below to auto-update this file in the end of the release cycle for the Java SDK:
https://github.com/dapr/java-sdk/blob/master/scripts/update_docs.sh